apps/client/src/app/publish/publish.component.html

```diff
@@ -101,7 +102,7 @@ <h1>Publish</h1>
           <div class="form-group">
             <label>Version</label>
             <div>
-              <select class="form-control" [(ngModel)]="options.templateVersion" (click)="templateVersionChanged()">
+              <select class="form-control" [(ngModel)]="options.templateVersion" [disabled]="true">
```

Comment:
This is still too restrictive. This should be something more like `[disabled]="igSpecifiesTemplate"` and
```
public get igSpecifiesTemplate() {
  return !!this.implementationGuide.extension.find(e => e.url === '...ig-template-type');
}
```